src/sentry/utils/sentry_apps/webhooks.py

```diff
@@ -145,10 +145,19 @@ def send_and_save_webhook_request(
         operation_type=SentryAppInteractionType.SEND_WEBHOOK, event_type=event
     ).capture() as lifecycle:
         buffer = SentryAppWebhookRequestsBuffer(sentry_app)
-
         org_id = app_platform_event.install.organization_id
         slug = sentry_app.slug_for_metrics
         url = url or sentry_app.webhook_url
+        lifecycle.add_extras(
+            {
+                "org_id": org_id,
+                "sentry_app_slug": sentry_app.slug,
+                "url": url or "",
```

Comment:
i think it's ok if url is `None` or falsey for whatever reason

Reply:
;-; this addition was per mypy's insistence that extras can only take `int | str`